src/frontend/js/toc.js

```diff
@@ -136,28 +138,31 @@ function buildTableOfContents() {
 
     if (typeof d2reader === 'object') {
         var items = d2reader.tableOfContents;
-        var has_structure = items.length > 1;
+        var has_structure = items.length > 0;
```

Comment:
With this change the three lines below are redundant - has_structure will already be true.